website/src/components/Submission/FileUpload/fileProcessing.ts

```diff
@@ -266,6 +275,10 @@ export class ExcelFile implements ProcessedFile {
         return this.originalFile;
     }
 
+    header(): Promise<string | null> {
+        return Promise.resolve(null);
+    }
```

Comment:
I find it a bit weird that this file also has a `header` that does nothing. It's an ExcelFile, as a user, you could assume that it gives you a proper header. -> I would rather not implement this function than have this dummy implementation. 

Do you think we can do something such that only the fasta files get the `header` method?